packages/fxa-auth-server/lib/luaScripts/setAccessToken_2.lua

```diff
@@ -1,11 +1,16 @@
 local userId = KEYS[1]
 local tokenId = KEYS[2]
 local token = ARGV[1]
-local ttl = ARGV[2]
+local limit = tonumber(ARGV[2])
+local ttl = ARGV[3]
```

Comment:
If you're adding an argument to ARGV, you want to name this `setAccessToken_3.lua`, ya?

Reply:
no, only `KEYS` requires the name change

Reply:
Duh.